opentelemetry-user-events-logs/examples/basic-logs.rs

```diff
@@ -21,9 +21,21 @@ fn init_logger() -> LoggerProvider {
 
 fn main() {
     // Example with tracing appender.
+    // Create a new tracing::Fmt layer to print the logs to stdout. It has a
+    // default filter of `info` level and above, and `debug` and above for logs
+    // from OpenTelemetry crates. The filter levels can be customized as needed.
+    let filter_otel = EnvFilter::new("info").add_directive("opentelemetry=off".parse().unwrap());
     let logger_provider = init_logger();
-    let layer = layer::OpenTelemetryTracingBridge::new(&logger_provider);
-    tracing_subscriber::registry().with(layer).init();
+    let otel_layer = layer::OpenTelemetryTracingBridge::new(&logger_provider);
+    let otel_layer = otel_layer.with_filter(filter_otel);
+
+    let filter_otel = EnvFilter::new("info").add_directive("opentelemetry=debug".parse().unwrap());
```

Comment:
rename to indicate this filter is for fmt layer, not otel.

Reply:
done